template/cordova/lib/prepare.js

```diff
@@ -172,6 +172,15 @@ function applyCoreProperties(config, manifest) {
         manifest.getVisualElements().setDisplayName(name);
     }
 
+    var description = config.description();
+    manifest.getProperties().setDescription(description);
+    if (description) {
```

Comment:
This breaks if say you add a description to config.xml and then remove it - the old description will remain in appxmanifest - we might have to revert back the old description in this case.